src/main/java/dev/walshy/sfmetrics/MetricsModule.java

```diff
@@ -0,0 +1,45 @@
+package dev.walshy.sfmetrics;
+
+import dev.walshy.sfmetrics.charts.AddonsChart;
+import dev.walshy.sfmetrics.charts.AutoUpdaterChart;
+import dev.walshy.sfmetrics.charts.CommandChart;
+import dev.walshy.sfmetrics.charts.CompatibilityModeChart;
+import dev.walshy.sfmetrics.charts.GuideLayoutChart;
+import dev.walshy.sfmetrics.charts.PlayerLanguageChart;
+import dev.walshy.sfmetrics.charts.ResearchesEnabledChart;
+import dev.walshy.sfmetrics.charts.ResourcePackChart;
+import dev.walshy.sfmetrics.charts.ServerLanguageChart;
+import dev.walshy.sfmetrics.charts.ServerSizeChart;
+import dev.walshy.sfmetrics.charts.SlimefunVersionChart;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import org.bstats.bukkit.Metrics;
+
+@SuppressWarnings("unused")
+public class MetricsModule {
+
+    private MetricsModule() {}
+
+    public static void start() {
+        Metrics metrics = new Metrics(SlimefunPlugin.instance(), 4574);
+
+        if (SlimefunPlugin.getUpdater().getBranch().isOfficial()) {
+            // We really do not need this data if it is an unofficially modified build...
+            metrics.addCustomChart(new AutoUpdaterChart());
+        }
+
+        metrics.addCustomChart(new ResourcePackChart());
+        metrics.addCustomChart(new SlimefunVersionChart());
+        metrics.addCustomChart(new ServerLanguageChart());
+        metrics.addCustomChart(new PlayerLanguageChart());
+        metrics.addCustomChart(new ResearchesEnabledChart());
+        metrics.addCustomChart(new GuideLayoutChart());
+        metrics.addCustomChart(new AddonsChart());
+        metrics.addCustomChart(new CommandChart());
+        metrics.addCustomChart(new ServerSizeChart());
+        metrics.addCustomChart(new CompatibilityModeChart());
```

Comment:
I don't know why you changed the visibility of all these classes to be public, I think they should remain "default", so package-private.

Reply:
Because you can't access them otherwise and I didn't want to dump everything in the same package. In an earlier iteration I also just searched the package and loaded all classes, no need for manual registration hence why the structure is this way. But, regardless, this is still cleaner and makes more sense.

Reply:
Ah okay, didn't see this was in a different package.